tests/Integration/Momento.Sdk.Incubating.Tests/DictionaryTest.cs

```diff
@@ -78,11 +79,11 @@ public async Task DictionarySetGetAsync_FieldIsByteArrayValueIsByteArray_NoRefre
         var field = Utils.NewGuidByteArray();
         var value = Utils.NewGuidByteArray();
 
-        CacheDictionarySetResponse setResponse = await client.DictionarySetAsync(cacheName, dictionaryName, field, value, false, ttl: TimeSpan.FromSeconds(5));
+        CacheDictionarySetResponse setResponse = await client.DictionarySetAsync(cacheName, dictionaryName, field, value, CollectionUpdateTtl.fromTimespan(TimeSpan.FromSeconds(5)).refreshTtlOnUpdates(false));
```

Comment:
This is confusing at first glance. It reads like you want to update the TTL, but then you don't. What it says, I think, is "set the TTL to 5 seconds, but don't refresh it".

I think there's confusion because "update" and "refresh" have very similar meaning. I think there's two things being conflated: the state of the collection and what action to take.

* Exists (creating a new collection)
  * Use my TTL
  * Use default TTL
* Does not exist (updating and existing collection)
  * Leave the TTL alone
  * Use my TTL
  * Use cache default TTL

So we'd have `onCreate*` and `onUpdate*` methods. The above might be...

```
CollectionTtl.onCreateSetTtl(someTimespan).onUpdateDontRefresh()
```

Reply:
Similarly `CollectionUpdateTtl`. What are we updating when the collection doesn't exist? Maybe just `CollectionTtl` or `CollectionUpdateTtls`?

Reply:
To set the TTL and use the same TTL on update...

```
CollectionTtl.onCreateSetTtl(someTimespan).onUpdateRefresh()
```

To set the TTL and use a *different* TTL on update...

```
CollectionTtl.onCreateSetTtl(someTimespan).onUpdateSetTtl(someOtherTimespan)
```

To use the default TTL on both...

```
CollectionTtl.onCreateUseDefault().onUpdateUseDefault()
```